### ISSUE TITLE
Bullet lists / Numbered lists don't break with 'Enter'

### ISSUE THREAD

**Issue by annvnzndrvn** [ckeditor/ckeditor5-angular#80]
When creating a new list, 'enter' key doesn't automatically add another bullet/number. However Ctrl+Enter works. Shift+Enter adds a '@'.

Does this need to be set up in a specific way so that it adds another bullet on 'enter' or is Ctrl+Enter the only way to go in the ckeditor5 angular version?

**Comment by ma2ciek** [comment]
Hi!

The error looks pretty weird. Could you describe how you got it? This is not related to the `ckeditor5-angular` package, it's only a wrapper. The problem is with your CKEditor5 build most probably.

**Comment by annvnzndrvn** [comment]
Cześć! :)

I've installed it using npm:

`npm i @ckeditor/ckeditor5-angular`

Added it to app.module.ts like this:

`import { CKEditorModule } from '@ckeditor/ckeditor5-angular';`

And inside imports like this:

```
imports: [
    BrowserAnimationsModule,
    CKEditorModule,
    etc...]
```

In my component ts file:

`import * as ClassicEditor from '@ckeditor/ckeditor5-build-classic';`

` public Editor = ClassicEditor;`


And finally in my component html view:

```
<ckeditor class="discussion-editor" 
    [editor]="Editor"
    [config]="{ toolbar: [ 'heading', '|', 'bold', 'italic', 'link', 'bulletedList', 'numberedList', 'blockQuote']}" 
    data="<p>lol!</p>">
</ckeditor>
```

I'm also hiding the .cke_top class like this in my global stylesheet:
```
.cke_top {
    display: none !important;
}
```

I hope this info helps!

**Comment by ma2ciek** [comment]
The integration looks pretty normal.

Could you add some more information which allows us to reproduce it? Maybe the browser version, system, etc. Or something you think it can change the behavior?

Also, it would be perfect if you could check it outside of the angular environment, probably the following sample would be perfect:

(you can just create an index.html file in the project root directory)
```html
<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta http-equiv="X-UA-Compatible" content="ie=edge">
	<title>Document</title>
</head>

<body>
	<div id="editor"></div>
	<script src="node_modules/@ckeditor/ckeditor5-build-classic/build/ckeditor.js"></script>
	<script>
		ClassicEditor.create( document.getElementById( 'editor' ), {
			config: { toolbar: [ 'heading', '|', 'bold', 'italic', 'link', 'bulletedList', 'numberedList', 'blockQuote' ] }
		} )
			.then( editor => window.editor = editor )
			.catch( err => console.log( err ) );
	</script>
</body>

</html>
```

**Comment by annvnzndrvn** [comment]
Ok, so I created this index.html and it's working as expected. The weirdest thing is that I tried now on my angular project again inside the component, without modifying anything at all (except for that file I created to test that out in project root directory), and now it actually works fine. No '@' and bullet list elements are breaking every time I hit enter.

I have no idea what's going on but it seems it's fine now.

**Comment by ma2ciek** [comment]
It looks like a super strange error.

Do you use some other rich text editing library in the project? Or something that might affect typing behavior? If no, it would be perfect if you could create a simple repository with a reproducible error. Otherwise, it's impossible for me to help you.

You can also test this behavior on this repository, you'd just need to clone https://github.com/ckeditor/ckeditor5-angular, install it and run `npm run start` script. Maybe it's somehow related to your environment.

**Comment by annvnzndrvn** [comment]
That was it! You made me realize I actually used a previous version of ckeditor some time ago in this project and I never removed the cdn link from Angular's index.html.

I removed this:
`  <script src="//cdn.ckeditor.com/4.10.1/standard/ckeditor.js"></script>
`
from index.html and it started working fine.

I added it back again and problems started happening again. I guess it was interfering somehow with the new ckeditor. Thanks a lot!
